src/scss/pages/countries.scss

```diff
@@ -1,10 +1,12 @@
 .countryResults {
     display: grid;
-    grid-template-columns: repeat(5, 1fr);
```

Comment:
If we are using `display: grid` we should not remove a template. More on that [here](https://css-tricks.com/snippets/css/complete-guide-grid/).